.github/workflows/build.yml

```diff
@@ -1,8 +1,14 @@
 name: build
 
 on:
-  push:
   pull_request:
+    branches:
+      - master
+      - develop
+  push:
+      
+  
+
```

Comment:
That's OK. But in this step you should also run tests even if there are no test have written.
So, please update the job like you have in another branch (we should cover all work related to CI in this step)